databricks.json

```diff
@@ -0,0 +1,3623 @@
+                    "example_values": [
+                        "abc123"
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "on poll",
+            "description": "Ingest tickets from Databricks",
+            "verbose": "The action will ingest alerts that have been triggered within Databricks",
+            "type": "ingest",
+            "identifier": "on_poll",
+            "read_only": true,
+            "parameters": {
+                "start_time": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored in this app",
+                    "order": 0
+                },
+                "end_time": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored in this app",
+                    "order": 1
+                },
+                "container_id": {
+                    "data_type": "string",
+                    "description": "Parameter ignored in this app",
+                    "order": 2
+                },
+                "container_count": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored in this app",
+                    "order": 3
+                },
+                "artifact_count": {
+                    "data_type": "numeric",
+                    "description": "Parameter ignored in this app",
+                    "order": 4
+                }
+            },
+            "output": [],
+            "versions": "EQ(*)"
+        }
+    ],
+    "pip_dependencies": {
+        "wheel": [
+            {
+                "module": "PyJWT",
+                "input_file": "wheels/py3/PyJWT-2.4.0-py3-none-any.whl"
+            },
+            {
+                "module": "certifi",
+                "input_file": "wheels/py3/certifi-2023.5.7-py3-none-any.whl"
+            },
+            {
+                "module": "charset_normalizer",
+                "input_file": "wheels/py3/charset_normalizer-2.0.12-py3-none-any.whl"
+            },
+            {
+                "module": "click",
+                "input_file": "wheels/py3/click-8.0.4-py3-none-any.whl"
+            },
+            {
+                "module": "databricks_cli",
+                "input_file": "wheels/py3/databricks_cli-0.17.6-py3-none-any.whl"
+            },
+            {
+                "module": "idna",
+                "input_file": "wheels/py3/idna-3.4-py3-none-any.whl"
+            },
+            {
+                "module": "importlib_metadata",
+                "input_file": "wheels/py3/importlib_metadata-4.8.3-py3-none-any.whl"
+            },
+            {
+                "module": "oauthlib",
+                "input_file": "wheels/py3/oauthlib-3.2.2-py3-none-any.whl"
+            },
+            {
+                "module": "requests",
```

Comment:
I believe that requests is being added as a sub-dependency of the cli, however since I'm on an M1 laptop when I do development I am first forced to use 'pypi' rather that wheels in my json file until I'm finished. I'm not sure if that is the reason why I am getting strange results in the wheel versions? I will try deleting all of the dependencies and re-generate them. I also saw that there is a new cli version so I'm going to upgrade that while I'm at it.